test/queryList.spec.js

```diff
@@ -1,49 +1,24 @@
-(function (root, factory) {
-    "use strict";
+const Stardog = require('../lib');
 
-    if (typeof exports === "object") {
-        // NodeJS. Does not work with strict CommonJS, but
-        // only CommonJS-like enviroments that support module.exports,
-        // like Node.
-        module.exports = factory(require("../js/stardog.js"), require("expect.js"));
-    } else if (typeof define === "function" && define.amd) {
-        // AMD. Register as an anonymous module.
-        define(["stardog", "expect"], factory);
-    } else {
-        // Browser globals (root is window)
-        root.returnExports = factory(root.Stardog, root.expect);
-    }
-}(this, function (Stardog, expect) {
-    "use strict";
+describe('queryList()', () => {
+  var conn;
 
-    describe ("List running queries", function() {
-        var conn;
+  beforeEach(() => {
+    conn = new Stardog.Connection();
+    conn.setEndpoint('http://localhost:5820/');
+    conn.setCredentials('admin', 'admin');
+    conn.setReasoning(true);
+  });
 
-        this.timeout(50000);
+  afterEach(() => {
+    conn = null;
+  });
 
-        beforeEach(function() {
-            conn = new Stardog.Connection();
-            conn.setEndpoint("http://localhost:5820/");
-            conn.setCredentials("admin", "admin");
-            conn.setReasoning(true);
-        });
-
-        afterEach(function() {
-            conn = null;
-        });
-
-        it ("should start a query and list it with the listQueries call.", function (done) {
-            conn.queryList(function (data, response) {
-                expect(response.statusCode).to.be(200);
-
-                expect(data).not.to.be(undefined);
-                expect(data).not.to.be(null);
-                expect(data.queries).not.to.be(undefined);
-                expect(data.queries).not.to.be(null);
-                expect(data.queries instanceof Array).to.be(true);
-                expect(data.queries.length).to.be(0);
-                done();
-            });
-        });
+  it('should start a query and list it with the listQueries call.', done => {
+    conn.queryList((data, response) => {
+      expect(response.statusCode).toEqual(200);
+      expect(data.queries).toHaveLength(0);
+      done();
```

Comment:
Have it kick off a long-enough-running query such that you can assert data.queries to have length of 1. This test currently is simply testing that queryList says there are 0 queries running when there are 0 queries running